app/models/concerns/search.rb

```diff
@@ -0,0 +1,10 @@
+module Search
+  extend ActiveSupport::Concern
+  module ClassMethods
+    def search(query)
+      lambda do |args|
+        where(query, "%#{args}%", "%#{args}%", "%#{args}%")
```

Comment:
In this method we get query like:

where("'email like ?'", "%#{args}%")

And as i understood in this way we are protected ffrom SQL injection:
http://gavinmiller.io/2015/fixing-sql-injection-vulnerabilities/
http://rusrails.ru/ruby-on-rails-security-guide#kontrmery

or not?